internal/server/data/migrations.go

```diff
@@ -460,3 +463,39 @@ DROP SEQUENCE IF EXISTS password_reset_tokens_id_seq CASCADE;
 		},
 	}
 }
+
+func addDefaultOrganization() *migrator.Migration {
+	return &migrator.Migration{
+		ID: "2022-08-10T13:35",
+		Migrate: func(tx *gorm.DB) error {
+
+			stmt := `
+INSERT INTO organizations(id, name, created_at, updated_at)
+VALUES (? , ?, ?, ?);
+`
+			orgID := uid.New()
+			now := time.Now()
+			if err := tx.Exec(stmt, orgID, "Default", now, now).Error; err != nil {
```

Comment:
We should make this "Default" name a constant similar to `models.InternalInfraProviderName`

Reply:
We definitely should, but I don't think we'd want to use that constant here. Migrations need to be frozen in time from the point they are initially released. We could hypothetically change the constant, but this migration should never change.